src/jira/client/axios.ts

```diff
@@ -63,7 +63,7 @@ const getErrorMiddleware = (logger: Logger) =>
 
 		// Log appropriate level depending on status - WARN: 300-499, ERROR: everything else
 		// Log exception only if it is error, because AxiosError contains the request payload
-		(isWarning ? logger.warn : logger.error)({ err: error, res: error?.response }, errorMessage);
+		logger[isWarning ? "warn" : "error"]({ err: error, res: error?.response }, errorMessage);
```

Comment:
I would find a simple `if/else` much easier on the eyes...

Reply:
done